src/NuGet.Services.EndToEnd/PushTests.cs

```diff
@@ -55,5 +55,32 @@ public PushTests(PushedPackagesFixture pushedPackages, ITestOutputHelper logger)
 
             await _clients.V3Search.WaitForPackageAsync(package.Id, package.Version, _logger);
         }
+
+        public async Task NewSemVer2PackageIsFilteredButAvailableInV3()
```

Comment:
Just focus on `Filtered` or `AvailableInV3` not both. I think you just care about `Filtered`